addons/website/models/website.py

```diff
@@ -83,6 +83,14 @@ def _default_language(self):
     default_lang_id = fields.Many2one('res.lang', string="Default Language", default=_default_language, required=True)
     auto_redirect_lang = fields.Boolean('Autoredirect Language', default=True, help="Should users be redirected to their browser's language")
     cookies_bar = fields.Boolean('Cookies Bar', help="Display a customizable cookies bar on your website.")
+    extended_cookies_consent = fields.Selection(
+        selection=[
+            ('no_choice', 'Only "I Agree"'),
```

Comment:
That's not asking for consent, that's annoying your users for no valid reasons. Ask or dont show anything.

Reply:
Hi @mart-e , 'Only I agree' is the only option available at the moment. 
We prefer to keep it because it was deemed important at some point.

Reply:
Note that this PR wasn't reviewed at all yet, it is based on a ~3 years old branch we were not allowed to merge back then, so I am just speaking about the 2 cookies bar mode.

> That's not asking for consent, that's annoying your users for no valid reasons.

We all agree about that.

It's not a matter of how you me or anyone else in Odoo see it but how our users do.

Thought I would summarize the rationale and history behind all this, as I think many Odoo people will be unhappy with this change and would prefer the cookies bar to simply disappear, not being "improved".
We are on the same page and would prefer an internet without cookies bar, we just realized that it was important to provide a solution for those people.

Our user base is requesting it again and again.
That was already the case a few years ago when we managed to convince FP to introduce a very light cookies bar despite it being useless and "people should know better than the GDPR". That's also why we have that weird looking confirm dialog when you enable the settings, it was the compromise back then to be allowed to merge it.

As a reminder, we were having hundreds of tickets back then, always telling people to introduce custom script and use https://cookie-bar.eu/ because "Odoo is smarter than GDPR, you don't need it, please read article XXX of the GDPR, they are wrong, you should involve a lawyer in case of problem, you don't need to comply to it".

I agree that we should try and educate people (as we do with the confirm modal), but the most important part is to satisfy the demand, when they are recurrent and there is no work around, especially when our concurrent does it.

Fact is, we shouldn't even care if this is correct or not regarding GDPR and cookies law policy. People don't know anything about it, they just know (or think) that they should have a cookie bar.
After all, your bank website, your town website, your government website, the police website, everyone has it.
It just became a standard (a shitty one we all agree but that's not the point), and people just want to (easily and quickly) be able to set one on their website, they don't care much more than that.

This one is just going a step further as there is still plenty of people not happy with the current cookies bar (while most are happy with it) and want it to be more granular.

The idea behind keeping both mode (the current one) was to keep a default cookie bar really light and minimalist, with a single "I agree" button and not impacting the business code (no need to check if cookie is set).
But indeed, we should probably just remove that new settings and somehow keep the default cookie bar layout as it is right now, with just an extra "Only essential" link :+1:** 
The less settings the better, and apart from having a lighter/simpler cookies bar, it doesn't really provide anything else, so making the new one also minimalist should be enough.

New mode (current PR):
![image](https://user-images.githubusercontent.com/30048408/187924497-41eb77e3-60e1-4d67-8cba-123cc38ebae4.png)
Old mode (without this PR):
![image](https://user-images.githubusercontent.com/30048408/187924698-3b25e969-b75e-4037-9885-0099c1987119.png)

What we could do (no mode settings):
![image](https://user-images.githubusercontent.com/30048408/187925980-43329f1c-46cc-4ff9-8963-da37cf5631e5.png)

At the end, it is exactly the same issue as for the Google Fonts and the people being sued in Germany, we didn't want to do anything at first, but we will introduce a workaround before end of the month for people to be able to download fonts locally and not use google server. Not point leaving people unhappy and thinking we are wrong. Especially when people are now sending scam attempt to website using google server for fonts. That would not be helping the product.

Reply:
> Note that this PR wasn't reviewed at all yet, it is based on a ~3 years old branch we were not allowed to merge back then

Sorry if commented too fast :) If this still need some work, maybe put it back in draft? I quickly tested and actually got analytic cookies stored _before_ I got to make a choice.

Thank you for the detailed explanation of the history (that I was not aware of). I don't like them but I get why we add the feature to have a cookie bar if people are asking for it. But let's offer a valid one then.
I think the "improved" cookie bar is the only valid one. The `Only "I accept"` should **not** be an option as it does not bring any advantage, the only two choices should be "No banner" or "Choice banner".
Also, the "Only essential" option should not be displayed lightly, in a way to favour the "Agree all", that's a shitty move that we see way too often...

Reply:
Agreed with PO: now only enabling GA only if consent is not an option or if consent was given.

Reply:
> Hi @mart-e , 'Only I agree' is the only option available at the moment.

Why not add an option for the site owner to disable optional cookies? Only set required cookies and show no bar.